app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt

```diff
@@ -412,7 +412,8 @@ object AchievementsModule {
                 "AddFerryAccessMotorVehicle", // 103
                 "AddInformationToTourism", // 137
                 "AddBoardType", // 188
-                "AddPostBoxRoyalCypher"
+                "AddPostBoxRoyalCypher",
+                "AddPoliceType"
```

Comment:
Is it really rare? How many police stations without operator are there in Italy? (Maybe the Overpass wizard @matkoniecz can help here :wink:)

I guess the same question can also be asked for `AddPostBoxRoyalCypher`, though.

Reply:
Yeah, very! In my area (Southern Italy) I think I'm the only one who adds operators to police. But, if we could have a official response that would be fantastic!

Reply:
It doesn't look so rare: https://overpass-turbo.eu/s/15p4

![grafik](https://user-images.githubusercontent.com/202916/112489072-63852d00-8d7e-11eb-8b50-1d82da473ed4.png)

It's 728 nodes (no ways or relations) matching your query in the displayed region alone. So I think this quest shouldn't count towards the "rare" achievement.

Reply:
`AddPostBoxRoyalCyper` is even worse though :sweat_smile: 

![grafik](https://user-images.githubusercontent.com/202916/112489828-1bb2d580-8d7f-11eb-9a44-b7f9004a98a0.png)
https://overpass-turbo.eu/s/15p9

6890 quests in the London region alone.

Reply:
Mh, maybe it's not as rare as I thought :cry: That was just my plan to get the achievement faster :smiling_imp: 
Just kidding, if everyone agrees I can remove also `AddPostBoxRoyalCyper`. For now I will remove mine from the achievement then. I just did the same query in my area and it's quite popular there too, I'm quite surprised, indeed the only ones that appear to lack the `operator` tag are the ones in rural areas or in areas where people do not live (near shopping centers and big ways).

Reply:
I don't know if it's me who hid the comment, I apologize if I did, that was not my intention!

Reply:
No, it was me; since we have more accurate numbers now, I've hidden my comment as outdated.

Reply:
Oh, perfect!